database/engine/rrdengineapi.c

```diff
@@ -361,11 +367,12 @@ void *rrdeng_get_page(struct rrdengine_instance *ctx, uuid_t *id, usec_t point_i
         return NULL;
     }
     *handle = descr;
+    pg_cache_descr = descr->pg_cache_descr;
 
-    return descr->page;
+    return pg_cache_descr->page;
 }
 
-void rrdeng_get_27_statistics(struct rrdengine_instance *ctx, unsigned long long *array)
+void rrdeng_get_28_statistics(struct rrdengine_instance *ctx, unsigned long long *array)
```

Comment:
Why not `rrdeng_get_statistics()` and just use proper definitions for the indexes below? (last entry could be RRD_ENG_STATISTICS_MAX or something)

Reply:
I want it to break whenever somebody decides to add a new statistic cause the risk is great.